about.html

```diff
@@ -1,68 +1,97 @@
-<!doctype html>
-
+<!doctype html>  
+  
 <html lang="en">
 <head>
-  <meta charset="utf-8">
+  <meta charset="utf-8">  
 
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
-</head>
+</head>  
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
+    <div class="container-about-page">
+      <div class="header">
+        <div class="logo">
+          <img src='img/lambda-black.png'>
+      </div>  
+        <nav>
+          <div>
+            <a class='home' href="index.html">Home</a>
```

Comment:
Good job linking back to the home page, Many people tend to forget this and it prevents the user of the website from properly navigating the site.